back/app/validators/css_color_validator.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# @see CssColorValidator#validate_each
+class CssColorValidator < ActiveModel::EachValidator
+  HEX_PATTERN = /^#[0-9a-fA-F]{3}([0-9a-fA-F]{3})?$/.freeze
+  HSL_PATTERN = /^hsl\(\s*(0|[1-9]\d?|[12]\d\d|3[0-5]\d)\s*,\s*((0|[1-9]\d?|100)%)\s*,\s*((0|[1-9]\d?|100)%)\s*\)$/.freeze
+  KEYWORDS = %w[none transparent inherit].freeze
```

Comment:
Should this array include the common color keywords, like 'black', 'blue', etc?

Maybe this is too verbose / detailed or maybe these color keywords are likely never used by our devs?